docs/CHANGELOG.md

```diff
@@ -0,0 +1,335 @@
+# Changelog
+
+## Version 2.13.1
+
+*January 31, 2023*
+
+* Fix Vagrantfile gem install for ruby >= 2.6 (thanks @Cyb0rk)
+* Disable file watcher in run_build() for sake of qemu on arm64 (thanks @anapsix)
+* Expand deprecated git.io links to full url in docs (thanks @judge2020)
+* Add margin to paragraph following code-block on phones (thanks @tlhunter)
+* Bump nokogiri from 1.13.4 to 1.13.9
+* Bump rouge from 3.28.0 to 3.30.0
+* Bump redcarpet from 3.5.1 to 3.6.0
+* Bump middleman from 4.4.2 to 4.4.3
+* Bump middleman-syntax from 3.2.0 to 3.3.0
+* Bump webrick from 1.7.0 to 1.8.1
+
+## Version 2.13.0
+
+*April 22, 2022*
+
+* __Drop support for ruby 2.5__
+* Bump rouge from 3.26.1 to 3.28.0
+* Formally support ruby 3.1
+* Bump nokogiri from 1.12.5 to 1.13.4
+* Build docker images for multiple architectures (e.g. `aarch64`)
+* Remove `VOLUME` declaration from Dockerfile (thanks @aemengo)
+
+The security vulnerabilities reported against recent versions of nokogiri should not affect slate users with a regular setup.
+
+## Version 2.12.0
+
+*November 04, 2021*
+
+* Bump nokogiri from 1.12.3 to 1.12.5
+* Bump ffi from 1.15.0 to 1.15.4
+* Bump rouge from 3.26.0 to 3.26.1
+* Bump middleman from 4.4.0 to 4.4.2
+* Remove unnecessary files from docker images
+
+## Version 2.11.0
+
+*August 12, 2021*
+
+* __[Security]__ Bump addressable transitive dependency from 2.7.0 to 2.8.0
+* Support specifying custom meta tags in YAML front-matter
+* Bump nokogiri from 1.11.3 to 1.12.3 (minimum supported version is 1.11.4)
+* Bump middleman-autoprefixer from 2.10.1 to 3.0.0
+* Bump jquery from 3.5.1 to 3.6.0
+* Bump middleman from [`d180ca3`](https://github.com/middleman/middleman/commit/d180ca337202873f2601310c74ba2b6b4cf063ec) to 4.4.0
+
+## Version 2.10.0
+
+*April 13, 2021*
+
+* Add support for Ruby 3.0 (thanks @shaun-scale)
+* Add requirement for Git on installing dependencies
+* Bump nokogiri from 1.11.2 to 1.11.3
+* Bump middleman from 4.3.11 to [`d180ca3`](https://github.com/middleman/middleman/commit/d180ca337202873f2601310c74ba2b6b4cf063ec)
+
+## Version 2.9.2
+
+*March 30, 2021*
+
+* __[Security]__ Bump kramdown from 2.3.0 to 2.3.1
+* Bump nokogiri from 1.11.1 to 1.11.2
+
+## Version 2.9.1
+
+*February 27, 2021*
+
+* Fix Slate language tabs not working if localStorage is disabled
+
+## Version 2.9.0
```

Comment:
Are all this files needed in the repo? Maybe the generate command should be enough? Then on the deployment server once the hook is received, triggers the build of the docs there and publishes. What do you think?

Reply:
This is exactly what I was trying to figure out now. I wasn't expecting such a blazing fast review!! I used slate. You can see how it looks like with current changes [on my fork here](https://hvitis.github.io/django-messages-drf/#introduction).

Basically there is just one file that changes. All docs is in 1 file called **index.html.md** so if there is any way to:
1. Install/clone slate in another place.
2. Replace the **index.html.md** file in slate executable folder 
3. Run build command
4. Switch branch to gh-pages and commit build result.

Then we are good. The out of the box deploy script that comes with slate needs .env file and accepts repo-link as origin variable to push the changes to gh-pages branch automatically.

Last commit shows that.